doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -109,9 +110,16 @@ The following best practices have been adopted to ensure verification and valida
 <a name="1.6"></a>
 ## 1.6 Verification methodologies
 
-Perform VNF interoperability verifications against an implementation of CNTT reference architecture, leveraging existing OPNFV Intake Process. Upstream projects will define features/capabilities, test scenarios, and test cases to augment existing OVP test harnesses to be executed via the OVP Ecosystem.
+Prior to onboarding a VNF the NFVi must be tested in Network, Compute and Storage domains to ensure that it provides required capabilities that allow the VNF to behave as expected.
+
+Predicting networking performance may be particularly difficult to establish and can be extremely sensitive to configuration details of the NFVi. Testing should be from the perspective of the VNF and create an environment that represents a realistic NFVi setup using open-source test tools and test workloads (i.e. test VNFs) that mimic how a real VNF will consume the NFVi. At this stage of testing we assume that the NFVi vendor has fully debugged and optimized the NFVi components and stack. With this approach the above objectives can be achieved:
+
+1. VNF vendors will know the capabilities of the NFVi and be able to forecast performance of the VNF. By knowing how the NFVi has been tested (and having the open-source test tools including workloads) they can reproduce test results or produce their own results using their own NFVi configurations.
+	
+2. Since testing is from the viewpoint of the VNF, performance of a commercial VNF should also be predictable. If this is not the case, the reference results provide good data to investigate the discrepancy. 
+	
+3. If the test tools are packaged as VNFs, they can be deployed using the same methods as a real VNF and hence are extremely portable across NFVi's.
 
-3rd Party test platforms may also be leveraged, if desired.
```

Comment:
Please consider adding this line back in, or some variant of, as it's intended to keep the option open for VNF(CNF) developers to utilize their own test harnesses, and tools, to complement the CNTT RC suite,  We were trying to be explicit that while badging would require specific RI-x/RC-x test suite execution, suppliers would be encouraged to perform additional testing using their tools and test cases for increased confidence of product behavior when given to a telco.

Reply:
How would this be done in practice?  Would VNF/CNF vendors create additional validation tools that run against the NFVI to validate assumptions?

Reply:
@kedmison - sure; VNF suppliers should be validating their VNF functionality using their own test harnesses, against their own labs/bench.  Therefore, inserting the statement, "if desired", is intended to encourage VNF suppliers to use their testing apparatus which includes their own scenarios to validate VNF functionality while using RI-x labs and pursing NFVI+VNF badging.  Keep in mind, our badging process does not include VNF functional testing.

Reply:
Similarly, @trevgc - can we revert/refactors some of the existing language so that we maintain the original intent of the chapter/paragraphs?